NetLogo 6.2.2
Tue May  2 23:25:41 EDT 2023
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 136 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 135 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 138 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 137 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 344 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 344 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 345 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 346 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 408 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 405 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 406 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 406 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 408 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 405 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 406 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 405 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 408 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 406 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 405 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 405 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 408 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 406 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 405 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 406 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 408 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 407 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 408 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 408 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 406 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 406 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Run #1, RUNTIME ERROR: MOVE-TO expected input to be an agent but got NOBODY instead.
error while seal 405 running MOVE-TO
  called by (anonymous command: [ x -> ask seals with [ x = label ] [ let mean [ xcor ] of seals with [ x = label ] let mean [ ycor ] of seals with [ x = label ] ifelse distancexy center_x center_y > 0.2 [ facexy center_x center_y forward 3.3 ] [ face one-of si-patches forward 3.3 ] ifelse distance one-of si-patches > 20 [ facexy 31 + random 8 50 + random 4 forward 3.3 ] [ move-to one-of si-patches set seals_home seals_home + 1 die ] if patch-here != ocean-patches [ move-to one-of ocean-patches in-radius 3 ] ] ])
  called by procedure MOVE_SEALS
  called by procedure GO
  called by procedure __EVALUATOR
org.nlogo.nvm.ArgumentTypeException: MOVE-TO expected input to be an agent but got NOBODY instead.
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:181)
	at org.nlogo.nvm.AnonymousCommand.perform(AnonymousProcedure.scala:158)
	at org.nlogo.prim.etc._foreach.perform(_foreach.scala:33)
	at org.nlogo.nvm.Context.runExclusive(Context.java:133)
	at org.nlogo.nvm.ExclusiveJob.run(ExclusiveJob.scala:39)
	at org.nlogo.nvm.Context.runExclusiveJob(Context.java:177)
	at org.nlogo.prim._asm_procedurego_ask_50.perform(:1)
	at org.nlogo.nvm.Context.stepConcurrent(Context.java:107)
	at org.nlogo.nvm.ConcurrentJob.step(ConcurrentJob.scala:65)
	at org.nlogo.job.JobThread.runPrimaryJobs(JobThread.scala:133)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:68)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
slurmstepd: error: *** JOB 6450 ON rocky6 CANCELLED AT 2023-05-03T23:25:48 DUE TO TIME LIMIT ***
